docs/reference/unit.md

```diff
@@ -446,3 +564,5 @@ associative, and symmetric under interchange of any inputs.
 
 - For _types_ `Us...`:
     - `CommonPointUnitT<Us...>`
+
+<script src="../assets/hrh4.js" async=false defer=false></script>
```

Comment:
It's supposed to add a horizontal rule between h4 sections, because they were otherwise too hard to distinguish visually. I did notice that it didn't seem to be working on this particular page, but I didn't have time to investigate. I decided to leave it in so I could communicate intent.